src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    public static void main(String[] args) {
+        Dog dog = new Dog();
+        Cat cat = new Cat();
+    }
+
+    private String getFromOneToTen() {
+        String result = "";
+        for (int i = 0; i < 10; i++) {
+            if (i % 2 == 0) {
+                result = result + i + " ";
+            }
+        }
+        return result;
+    }
+
+    private void sayHello(String hello) {
+        System.out.println("Hello, Mate! I am happy to see you! This "
+                + "class looks weird! Your task is to fix the checkstyle in this file."
+                + " Your goal is to run `mvn clean package` and receive `BUILD SUCCESS` message");
+        System.out.println(hello);
+    }
+
+    private void initializeVariables() {
+        String a = "Hello mates!";
+        System.out.println(a);
+        int variable = LocalDate.now().getYear();
```

Comment:
`a`, `variable` - uninformative variables names

Reply:
because my goal was to receive “BUILD SUCCESS” message.
https://prntscr.com/1najz22

Reply:
I fix it!